.github/workflows/test-build-deploy.yml

```diff
@@ -278,91 +281,120 @@ jobs:
       - name: Build Mimir with race-detector
         run: |
           export IMAGE_TAG_RACE=$(make image-tag-race)
-          export MIMIR_IMAGE="grafana/mimir:$IMAGE_TAG_RACE"
+          # We are working on migrating to a distroless image as our only base.
+          # For now we mark our distroless image with a tag prefix `distroless-`
+          # and Alpine remains the default image pushed to the registry.
+          # We always test the distroless image, and we test the alpine image on deploy.
+          export MIMIR_IMAGE="grafana/mimir:distroless-$IMAGE_TAG_RACE"
+          export MIMIR_ALPINE_IMAGE="grafana/mimir:$IMAGE_TAG_RACE"
```

Comment:
Since these env vars are not used by `make` and only in subsequent `docker save` commands, we should just inline them. Now it looks like `make` uses them somehow, which is confusing.